.github/workflows/main.yml

```diff
@@ -36,12 +36,13 @@ jobs:
       - name: create zip package
         id: create-package
         env:
-          file_hash: ${{ hashFiles('app/**', 'requirements.txt') }}
+          file_hash: ${{ hashFiles('app/**', 'requirements.txt', 'handlers/**') }}
         run: |
           pip3 install -r requirements.txt --target dependencies
           cd ./dependencies && zip -r9 ../$file_hash .
           # add APP files to Zip file
           cd .. && zip -g ./$file_hash -r ./app
+          zip -ur ${file_hash}.zip handlers
```

Comment:
in next PR restructure this packaging process

copy all files into a directory and then zip it once